protocols/pcr_prep/pcr_prep.py

```diff
@@ -0,0 +1,868 @@
+    ctx.comment("and the target plate in slot {} ({})"
+                .format(lw_slot_dict["target_plate_step_2"],
+                        _DNA_sample_plate))
+    if not _use_same_pipettes:
+        ctx.comment("Change the pipettes for step 2, left: {}, right: {}"
+                    .format(_left_pipette_part_2, _right_pipette_part_2))
+    ctx.pause()
+
+    if _create_mastermix:
+        del ctx.deck[4]
+        del ctx.deck[7]
+        del ctx.deck[10]
+        temp_mod_slot_4, temp_mod_slot_7, temp_mod_slot_10 = None, None, None
+        load_temp_mods()
+
+    # We have to repopulate the dictionary since Python assigns by value.
+    temp_mod_dict = \
+        {"mastermix": temp_mod_slot_4,
+         "pcr_reagent_labware_step_1": temp_mod_slot_7,
+         "aux_pcr_reagent_labware_step_1": temp_mod_slot_10,
+         "DNA_sample_plate_step_2": temp_mod_slot_7,
+         "target_plate_step_2": temp_mod_slot_10
+         }
+
+    mastermix_label = "Mastermix container"
+    dna_label = "DNA sample plate"
+    target_label = "DNA/Mastermix target plate"
+
+    dna_sample_plate = None
+    mastermix_labware = None
+    dna_mastermix_target_plate = None
+
+    # Load mastermix container
+    if temp_mod_dict["mastermix"] is not None:
+        mastermix_labware = \
+            temp_mod_dict["mastermix"].load_labware(_mastermix_labware)
+    else:
+        mastermix_labware = \
+            ctx.load_labware(_mastermix_labware, lw_slot_dict["mastermix"],
+                             mastermix_label)
+    # Load DNA sample plate
+    if temp_mod_dict["DNA_sample_plate_step_2"] is not None:
+        dna_sample_plate = \
+            (temp_mod_dict["DNA_sample_plate_step_2"].
+                load_labware(_DNA_sample_plate, dna_label))
+    else:
+        dna_sample_plate = \
+            ctx.load_labware(_DNA_sample_plate,
+                             lw_slot_dict["DNA_sample_plate_step_2"],
+                             dna_label)
+
+    # Load the target plate
+    if temp_mod_dict["target_plate_step_2"] is not None:
+        dna_mastermix_target_plate = \
+            temp_mod_dict["target_plate_step_2"].load_labware(_target_labware,
+                                                              target_label)
+    else:
+        dna_mastermix_target_plate = \
+            ctx.load_labware(_DNA_sample_plate,
+                             lw_slot_dict["target_plate_step_2"],
+                             target_label)
+
+    # Distribute mastermix
+    ctx.comment("\nDistributing mastermix into column 1 of the target plate\n")
+    mastermix_source = mastermix_labware.columns()[0]
+
+    n_target_columns = floor(_number_of_samples / 8)
+    remainder = _number_of_samples % 8
+
+    # Pipette mastermix into the first column of the target,
+    # then distribute through the rows
+
+    # Drop any tips
+    drop_all_tips([left_pipette, right_pipette])
+
+    def SCP_distribute_mastermix(pipette, source_well, dest_plate):
```

Comment:
not a huge deal, but typically [we don't want to start the name of our function with a capital letter.](
https://www.python.org/dev/peps/pep-0008/#function-and-variable-names)